pages/css/architecture.md

```diff
@@ -3,36 +3,42 @@ title: Architecture
 permalink: /css/architecture/
 sidenav: css
 ---
-# Architecture
+# Choosing a CSS methodology and architecture
 A site's architecture should be based on its goals and purposes. This means the
 guidance here should be adapted to different sites and situations.
 
-## Modular or component architecture
-When using a modular or component architecture, every page is broken into a
-series of modular components. There are two sets of these components:
-`components` and `modules`. The architecture starts out with basic HTML element
-rules: HTML, p, a, form, etc tags that than have components and modules written
-on top of them. Components are very basic structure such as buttons, blurbs,
-navs, and positioning structures like insets, island, and enclosure.  From
-here, modules are built with these components. This architecture also attempts
-to keep the specificity trend in an upwards curve as you move down in the file
+In general, 18F recommends using a modular approach (specifically, [modified BEM])
+to provide consistency and scalability to larger apps. However, for some applications
+(such as a rapid prototype), an atomic architecture may make more sense. Below are 
+some considerations for choosing an architecture.
+
+## Modular architecture
+Naming conventions like [BEM] and [SMACSS] popularized the idea of modular architecture.
+Modular architectures are very helpful for a few different kinds of projects:
+- larger, more complex sites with reusable components
+- sites maintained and built by multiple engineers
+- production sites that will be handed off to partner teams or vendors
+
+In a modular architecture, every page is broken into `components` and `modules`.
+*Components* provide basic recurring elements such as buttons, lists, and navs. 
+*Modules* are built by combining these components into reusable structures.
+This helps keep the specificity trend in an upwards curve as you move down in the file
 (more on this to come).
 
-- Start with an elements file for all tag rules (a, h1-h5, p, \*, html, body).
+A few helpful things to keep in mind when building out a modular architecture:
+- Start with an elements file for all tag rules (`a`, `h1`-`h5`, `p`, `*`, `html`, `body`).
```

Comment:
Why? 

I don't think it's a bad idea, it just feels like a fairly strong opinion to not have a reason.